scripts/visualize_size.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2020 Datadog, Inc.
+
+# Publish the datadog lambda layer across regions, using the AWS CLI
+# Usage: VERSION=5 REGIONS=us-east-1 publish_layers.sh
+# VERSION is required.
+set -e
+
+# Move into the root directory, so this script can be called from any directory
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd $DIR/..
+
+./serverless/build_binary_and_layer.sh
```

Comment:
Did you start your branch after Nick's PR about refactoring build scripts ? It seems that was the name of the old folder

Reply:
Fixed!